app/controllers/users/webauthn_setup_controller.rb

```diff
@@ -183,17 +183,12 @@ def process_invalid_webauthn(form)
         else
           flash.now[:error] = t('errors.webauthn_setup.unique_name')
         end
-
-        render :new
-      else
-        if form.platform_authenticator?
-          flash[:error] = t('errors.webauthn_platform_setup.general_error')
+      elsif form.platform_authenticator?
+        flash[:error] = t('errors.webauthn_platform_setup.general_error')
         else
           flash[:error] = t('errors.webauthn_setup.general_error')
```

Comment:
Can we fix the indentation here? (Surprised the linter doesn't catch it)

```suggestion
      else
        flash[:error] = t('errors.webauthn_setup.general_error')
```